src/main/webapp/index.html

```diff
@@ -19,9 +19,17 @@
     <script type="text/javascript" charset="utf-8" src="js/fruzhin.js"></script>
     <script type="text/javascript" charset="utf-8" src="js/http.js"></script>
 </head>
-<body onload="main()">
+<body>
 <script>
-    let start = async () => {
+    var libp = undefined;
+
+    var getPeerId = () => {
+        return libp?.peerId;
+    }
+
+    var start = async (bootnodes) => {
```

Comment:
Where do we populate the bootnodes list?

Reply:
It's fetched from chainspec